api/src/resolvers/banking/banking-cypher.ts

```diff
@@ -54,3 +54,26 @@ DELETE r
 
 RETURN record, church.name AS churchName, date.date AS date
 `
+export const lastButOneServiceRecord = `
+MATCH (record:ServiceRecord {id: $serviceRecordId})
+MATCH (record)<-[:HAS_SERVICE]-(:HistoryLog)<-[:HAS_HISTORY]-(fellowship) 
+WHERE fellowship:Fellowship OR fellowship:ClosedFellowship
```

Comment:
Is there a need for searching closed fellowship?

Reply:
If the closed fellowships show up on the church list for banking, there is a need.